### PR TITLE
add language section list

### PR DESCRIPTION
**Pull request by Rahul-Kumar-prog**
This is the Result 
Screenshot 2022-10-07 at 6 48 14 PM

### CODE REVIEW

**Comment by Rahul-Kumar-prog** [comment]
Hey @max-programming please review my PR and let me know if need any changes on it. 
Thankyou

**Comment by Rahul-Kumar-prog** [comment]
And the latest update of this site have some issue i recognise it. if you allow then we can discuss about this

**Comment by max-programming** [comment]
@Rahul-Kumar-prog Thanks for this! 
I think it's good to separate both dropdowns

**Comment by Rahul-Kumar-prog** [comment]
@max-programming you mean language with icon?
and can you explain little bit more about (separate both drop-downs on different lines)

**Comment by max-programming** [comment]
@Rahul-Kumar-prog I mean both of them should be on separate lines instead of the same line
image

**Comment by max-programming** [comment]
For the icon, you can add any icon that could make them look a little different. Maybe a **"coding"** icon on the language and a **"sort"** icon on the other one

**Comment by Rahul-Kumar-prog** [comment]
Do you mean the sort list under the language list?
and need to add icons instead of the name or need to add icons with names?

**Comment by max-programming** [comment]
Yes. And icons with the name is better

**Comment by Rahul-Kumar-prog** [comment]
ok I will do it thanks

File: components/Sort.tsx [hunk 1]
```diff
@@ -32,7 +37,29 @@ export default function Sort() {
     }
   };
   return (
-    <div className="flex justify-center mb-2">
+    <div className="flex justify-center mb-2 flex gap-2">
```
Comment (Dun-sin) [inline components/Sort.tsx:40]:
```suggestion
    <div className="flex justify-center items-center mb-2 flex-col gap-2">
```

this should fix it

Comment (max-programming) [reply components/Sort.tsx]: Thanks a lot @dun-sin

**Comment by max-programming** [comment]
The only thing left now is the icons @Rahul-Kumar-prog

**Comment by Rahul-Kumar-prog** [comment]
@Dun-sin big thx for helping me thankyou

**Comment by Rahul-Kumar-prog** [comment]
@max-programming can you suggest me the sort icon

**Comment by Rahul-Kumar-prog** [comment]
Screenshot 2022-10-13 at 3 45 10 PM

Have a look on it 
i made changes for the wright or the dropdown list

**Comment by max-programming** [comment]
> @max-programming can you suggest me the sort icon
> 
> Screenshot 2022-10-13 at 3 54 38 PM
> what about this?

This sort icon looks better to me.

**Comment by Rahul-Kumar-prog** [comment]
Hey @max-programming you merged this pr ? so I have to ask if i am changing the icon of the language and sort. i have to create a new issue about that or I can work with pr same.
and One more thing is fixing the reset problem may have taken some time. is it ok for taking time?

One most important thing is when i working and stuck in a problem and go on Twitter for help i know that the project use Tailwind CSS but in readme or language use section does not mention about that so its good to create an issue about that

**Comment by max-programming** [comment]
@Rahul-Kumar-prog I haven't merged this PR yet but don't worry. I will merge it now. For the icons, I will handle it

**Comment by Rahul-Kumar-prog** [comment]
i added icons to it. should i push it.
Screenshot 2022-10-15 at 1 20 35 PM

**Comment by max-programming** [comment]
Sure! Go ahead

**Comment by max-programming** [comment]
@Rahul-Kumar-prog For some reason i can only see Python in the list of languages

**Comment by max-programming** [comment]
Still the same case. Let me know when it works 😊

**Comment by max-programming** [comment]
@all-contributors please add @Rahul-Kumar-prog for code, design

**Pull request merged**